python_examples/simulation_flow.py

```diff
@@ -10,26 +10,26 @@
 
 # Method to set the seed at the start of the simulation, for reproducibility
 
-pe.routine.Simulation.set_random_seed(seed=42)
+# pe.routine.Simulation.set_random_seed(seed=42)
 
 # Pop_params are used to configure the population structure being used in this
 # simulation.
 
-pop_params = {"population_size": 100, "cell_number": 1,
-              "microcell_number": 1, "household_number": 20,
+pop_params = {"population_size": 1000, "cell_number": 10,
+              "microcell_number": 1, "household_number": 5,
               "place_number": 2}
 
 pe.Parameters.instance().time_steps_per_day = 1
-pe.Parameters.instance().base_reproduction_num = 2.8
+pe.Parameters.instance().basic_reproduction_num = 2.8
 
 # Create a population based on the parameters given.
 population = pe.routine.ToyPopulationFactory().make_pop(pop_params)
-cell = population.cells[0]
+[cell.set_location((i, 0)) for i, cell in enumerate(population.cells)]
```

Comment:
If you need cell locations there are specific methods to set this in `toy_population_config`